.github/ISSUE_TEMPLATE/android-issue.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Android Issue
 about: Template for logging issues on Android
-title: ''
-labels: ''
+title: "[Android] "
+labels: android-related
 assignees: ''
 
```

Comment:
nit: probably should remove this extra space.

Reply:
I think the intention is so that it has `[Android] ` in the title and then anything you put has the space between that and the actual title. Similar to https://github.com/brave/brave-browser/issues/7670

Reply:
whoops - I missed which type of whitespace you're talking about. You meant newline, not the space under `title`. ++ on removing that

Reply:
@bsclifton was referring to the space between `assignees: ''` and `---` on `Line 7`.

Reply:
Line 7 is automatically added by the template not by me intentionally. Just added the two entries for title and labels. I don't want to mess with the template it may just cause issues

Reply:
Gotcha 👍 Thanks @srirambv.